chainerx_cc/chainerx/routines/activation.cc

```diff
@@ -0,0 +1,21 @@
+
+#include <vector>
+
+#include "chainerx/array.h"
+#include "chainerx/constant.h"
+#include "chainerx/device.h"
+#include "chainerx/dtype.h"
+#include "chainerx/kernels/math.h"
+#include "chainerx/routines/activation.h"
+#include "chainerx/routines/creation.h"
+#include "chainerx/routines/explog.h"
+#include "chainerx/routines/indexing.h"
+#include "chainerx/scalar.h"
+
+namespace chainerx {
+Array Elu(const Array& x, Scalar alpha) {
+    Array zero = ZerosLike(x, x.device());
+    return Where(x > zero, x, alpha * Expm1(x));
```

Comment:
`x > zero` could be replaced with `x > 0` if `operator>` supports scalars.
Could you write a TODO?